README.md

```diff
@@ -98,6 +98,20 @@ Convert an observation to an OSM type `node`.
 
 Retrieves the current device's unique ID (string).
 
+### `status = mapeo.getDatabaseStatus()`
+
+Retrieves the status of the database, returns an array of hypercore feed
+download status.
+
+```
+[{
```

Comment:
@okdistribute What do you think about this being a map on the outside, so we have room later to extend for indexing status?

```js
{
  devices: [ ... ]  // same as you have here
  indexes: { ... }  // content to-be-determined
}
```

Reply:
Alternately, we could have separate `getDevicesStatus()` and `getIndexesStatus()` APIs. What do you think?

cc @gmaclennan

Reply:
We could do both :)

Reply:
I prefer separate functions (`getDevicesStatus()` and `getIndexesStatus()`) so that the client can choose what they need. I'm not a fan of both, because it adds API surface area without adding functionality, just convenience (just calling one method instead of two). If we eventually make the API promise based it becomes easy:

```js
const [devices, indexes] = await Promise.all(getDevicesStatus(), getIndexesStatus())
```

Reply:
I think either will work well enough. I feel good about @okdistribute just making the call based on our arguments versus us bikeshedding it. :+1:

Reply:
thanks for the input all,

because this is a small api, I'm going to go with the simple approach I had in the beginning, but rename to `getFeedStatus` to be more clear we're talking about the feeds (rather than hypercores?). When we add media and indexing support, we can add those functions as well